components/card/package.json

```diff
@@ -30,6 +31,7 @@
     "@spectrum-css/component-builder": "^1.2.0",
     "@spectrum-css/icon": "^2.1.1",
     "@spectrum-css/quickaction": "^3.0.0-beta.3",
+    "@spectrum-css/typography": "^2.1.3",
```

Comment:
Since this is in deps, we don't need it in devDeps too
```suggestion
```

Reply:
That's `peerDependencies`, not `dependencies`.